mail-connector/src/test/java/eu/tsystems/mms/tic/testframework/mailconnector/test/MailConnectorTest.java

```diff
@@ -472,7 +473,7 @@ public void testT07_signMessage() {
     public void testT08_sendAndWaitForMessageWithoutAttachement_SubjectSenderRecipient() throws Exception {
 
         final String subject = "testT08_sendAndWaitForMessageWithoutAttachement_SubjectSenderRecipient"
-                + StringUtils.getRandomStringWithLength(5);
+                + RandomStringUtils.random(5, true, false);
```

Comment:
Changed behaviour: Old method would only generate `[A-Z]^length`. (JavaDoc seems to be wrong?)
Sanity Check: Intended?

... The old behaviour was like this even in the first commit, so the documented result might have been the expected one to begin with...